src/autorenew-invite/index.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+import ovhAngularPaymentMethod from '@ovh-ux/ng-payment-method';
+import translate from 'angular-translate';
+import translateAsyncLoader from '@ovh-ux/translate-async-loader';
+
```

Comment:
Should import `ovh-angular-user-pref`  and declare it as module dependency.

Same for `ovh-ui-angular`